Frontend/ui-driver/src/Services/Backend.purs

```diff
@@ -489,7 +492,30 @@ makeUpdateDriverLangChange deviceToken = UpdateDriverInfoReq {
             "ML_IN" -> "MALAYALAM"
             "BN_IN" -> "BENGALI"
             "TA_IN" -> "TAMIL"
-            _       -> "ENGLISH"
+            _       -> "ENGLISH",
+        "gender"        : Nothing
+    }
+
+makeUpdateDriverGender :: String -> UpdateDriverInfoReq
```

Comment:
We should not create multiple functions to create update profile request body. Can you replace all the functions with a single mkUpdateProfileReq function?

For approach we can discuss together